test/basic-querying.test.js

```diff
@@ -153,6 +153,18 @@ describe('basic-querying', function() {
           done();
         });
     });
+    it('should query by ids and null condition', function(done) {
+      User.findByIds([
+        createdUsers[0].id,
+        createdUsers[1].id],
+        {where: {vip: null}}, function(err, users) {
```

Comment:
@paulussup @ebarault I am not sure if this test case checks for the fix in the right away. The fix is however correct.

Shouldn't the filter do `near` query with a null value and see if the null value it is taken care of? Something like this test case but with a `null` value filter: https://github.com/strongloop/loopback-datasource-juggler/commit/64f64caaff70898281b09d8469db4e7a89fa160f#diff-8fae24fa21c82e23f7f43ccdae4fcf90R599